ref(dynamic-sampling): Add missing custom inbound filters

priscilawebdev opened the pull request:
Adds the following missing custom options:

![image](https://user-images.githubusercontent.com/29228205/119500546-36e29700-bd68-11eb-8f4d-34e12ee64302.png)

- Ip Addresses - `event.client_ip` 
- Error Messages - `event.error_messages` 
- Content Security Policy - `event.csp` 

closes: https://app.asana.com/0/1182975495223897/1200315225518971

markstory reviewed (commented):
Frontend code changes look good to me.

RaduW reviewed (approved):
The part that adds missing inbound filters looks correct.

I didn't see any tests added though.

Also some minor unrelated changes (like changing some message) and also some code that seemed unrelated to the PR `handleChangeTransaction()` in `form.tsx` (didn't check what's happening there)
We shouldn't sneak in unrelated changes (although I also do that all the time).

priscilawebdev commented:
@RaduW good points!

I'll add tests as soon as I have some time. It's on my list 😉

In this PR I also updated that description we discussed offline and fixed one minor bug related to the `handleChangeTransaction` function

Sorry I thought that was fine to push these changes together, since they are minor. I'll try to not do that next time
